db/schema.rb

```diff
@@ -47,7 +47,6 @@
     t.date "average_views_updated_at"
     t.integer "wiki_id"
     t.integer "mw_page_id"
-    t.boolean "tracked", default: true
```

Comment:
The changes to schema.rb are not related to the PR and should not be included. (They are artifacts of the local settings at the time the migrations are run; unless we are making changing to the database tables, we usually avoid any changes to the version of schema.rb in the repo.)

Reply:
> I don't think we need to be concerned about duplication or already-existing pages for this feature... in many cases, the purpose of changing it will be to make the 'sandbox draft' link point to an already-existing page where the student is already underway with creating a draft.
> 
> The basic strategy you describe for the implementation so far makes sense. I think for the UI, the 'edit' link needs to be smaller so that it's more clear that it's not just another link but is something related to the sandbox draft link. The 'Change' link for the wiki language and project selector (eg, at https://dashboard.wikiedu.org/article_finder ) might be a good model to follow.

Thanks alot for the review, I'll get working on it immediately.

Reply:
> The changes to schema.rb are not related to the PR and should not be included. (They are artifacts of the local settings at the time the migrations are run; unless we are making changing to the database tables, we usually avoid any changes to the version of schema.rb in the repo.)

I'll fix this immediately. I always make sure to filter out all git noise. Sorry this slipped through.